openwisp_controller/connection/api/urls.py

```diff
@@ -20,6 +20,16 @@ def get_api_urls(api_views):
             api_views.command_details_view,
             name='device_command_details',
         ),
+        path(
+            'api/v1/connection/credential/',
+            api_views.credential_list_create_view,
+            name='credential_list',
+        ),
+        path(
+            'api/v1/connection/credential/<str:pk>/',
+            api_views.credential_detail_view,
```

Comment:
| SN | /api/v1/connection/credential/str:pk/          | Status | Details |
|:--:|------------------------------------------------|:------:|:-------:|
|  1 | [SU][O1A][O1O] Can view only appropriate creds |  Pass  |         |
|  2 | [SU][O1A][O1O] Can edit only appropriate creds |  Pass  |         |
|  3 | [SU][O1A] Correct data is shown                |  Pass  |         |
|  4 | [SU][01A] Can delete only appropriate data     |  Pass  |         |